### PR TITLE
Triplet check

### PR DESCRIPTION
**Pull request by daschuer**
This adds an early exit if the target triplet was not found. This happens if you try to install on a target currently not supported in the installed vcpkg environment. 

Now it fails with: 
```
CMake Error at CMakeLists.txt:109 (message):
  VCPKG_TARGET_TRIPLET dir not found: /Users/runner/buildenv/mixxx-deps-2.4-x64-osx-dynamic-min10.12-7756933/installed/x64-osx-dynamic-min10.12
```

Without this patch we have this output: 
```
CMake Error at /Users/runner/hostedtoolcache/cmake/3.16.9/x64/cmake-3.16.9-Darwin-x86_64/CMake.app/Contents/share/cmake-3.16/Modules/FindPackageHandleStandardArgs.cmake:146 (message):
  Could NOT find Chromaprint (missing: Chromaprint_LIBRARY
  Chromaprint_INCLUDE_DIR)
Call Stack (most recent call first):
  /Users/runner/hostedtoolcache/cmake/3.16.9/x64/cmake-3.16.9-Darwin-x86_64/CMake.app/Contents/share/cmake-3.16/Modules/FindPackageHandleStandardArgs.cmake:393 (_FPHSA_FAILURE_MESSAGE)
  cmake/modules/FindChromaprint.cmake:68 (find_package_handle_standard_args)
  /Users/runner/buildenv/mixxx-deps-2.4-x64-osx-dynamic-min10.12-7756933/scripts/buildsystems/vcpkg.cmake:824 (_find_package)
  CMakeLists.txt:1846 (find_package)
```

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -104,6 +104,12 @@ if(APPLE)
   endif()
 endif()
 
+if(DEFINED VCPKG_ROOT AND DEFINED VCPKG_TARGET_TRIPLET)
+  if(NOT EXISTS "${VCPKG_ROOT}/installed/${VCPKG_TARGET_TRIPLET}")
+    message(FATAL_ERROR "VCPKG_TARGET_TRIPLET dir not found: ${VCPKG_ROOT}/installed/${VCPKG_TARGET_TRIPLET}")
```
Comment (Swiftb0y) [inline CMakeLists.txt:109]: this is still very opaque for people that don't know what this means.

Comment (daschuer) [reply CMakeLists.txt]: Ok, I have added more context.
Is is OK now?

**Review by Swiftb0y — APPROVED** [review]
Thanks

**Comment by daschuer** [comment]
I had messed up line breaks. Now it looks like this: 
```
VCPKG_TARGET_TRIPLET dir not found: /Users/runner/buildenv/mixxx-deps-2.4-x64-osx-dynamic-min10.12-7756933/installed/x64-osx-dynamic-min10.12
Make sure the VCPKG build environment is installed and contains the build for the selected triplet
```

**Pull request merged**